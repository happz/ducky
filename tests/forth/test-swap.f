5 10 SWAP . . CR

