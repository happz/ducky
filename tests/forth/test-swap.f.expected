Ducky Forth welcomes you

5 10 
